content/building/environment-variables.md

```diff
@@ -23,11 +23,16 @@ Here you'll find some of the read-only environment variables explained.
 |--------------------------|-----------------------------------------------------------------------------------------------------------------|
 | CI                       | true                                                                                                            |
 | CONTINUOUS_INTEGRATION   | true                                                                                                            |
-| BUILD_NUMBER             | Number of the build for this project in Codemagic                                                               |
+| BUILD_NUMBER             | Number of the build for this project in Codemagic for the given workflow                                        |
+| PROJECT_BUILD_NUMBER     | Number of the build for this project in Codemagic                                                               |
 | FLUTTER_ROOT             | Absolute path to Flutter SDK                                                                                    |
-| FCI_BRANCH               | The current branch being built, for pull requests it is the destination branch                                  |
+| FCI_BRANCH               | The current branch being built, for pull requests it is the source branch                                       |
+| FCI_REPO_SLUG            | The slug of the repository that is currently being built in the form owner_name/repository_name. Unset in case of custom source added repository|
 | FCI_COMMIT               | Commit hash that is currently being built by Codemagic, for pull request builds it is the hash of the source commit |
-| FCI_CLONE_UNSHALLOW      | If set to `true`, performs a full clone of the repository instead of top 50 commits                              |
+| FCI_PULL_REQUEST         | `true`, if the current build is building a pull request, `false` otherwise                                      |
+| FCI_PULL_REQUEST_NUMBER  | Set to Integer ID of the pull request for the Git provider (Bitbucket, Github etc) if the current build is building a pull request, unset otherwise|
+| FCI_CLONE_UNSHALLOW      | If set to `true`, performs a full clone of the repository instead of top 50 commits                             |
+| FCI_CLONE_DEPTH          | Specifies the clone depth which allows to decrease default fetching time when is set lower than 50              |
```

Comment:
Specifies the number of commits to be fetched from the repository when cloning, default: 50. Specifying a smaller number can decrease the default fetching time.